_posts/2023-09-19-compiling-ml-models.md

```diff
@@ -1043,7 +1043,7 @@ NUM_PIXELS = 28*28
 NUM_DIGITS = 10
 inp = [Value(0, (), "input") for _ in range(NUM_PIXELS)]
 exp = [Value(0, (), "input") for _ in range(NUM_DIGITS)]
-out = model(inp)  # produces some garbage data in the compile-time Value graph
+out = model(inp)  # create the compile-time Value graph (with contains garbage data since inp is garbage input)
```

Comment:
s/with/which

and can you make sure this comment fits in 80 columns? otherwise it looks bad on web. feel free to use the next line as well